tool/site/client/src/pages/sendPreSignPage.tsx

```diff
@@ -53,20 +83,36 @@ const SendPreSignPage: React.FC<SendPreSignPageProps> = ({
   return (
     <LongRunningTask<Bundle> task={retrievePrescriptionTask} loadingMessage="Retrieving prescription details.">
       {bundle => {
-        if (!sendConfirmed) {
-          const summaryViewProps = createSummaryPrescription(bundle)
+        if (!sendPageFormValues) {
```

Comment:
Almost wondering if it might be easier to just always use the edit form and just pre-poulate the form fields with the default values? then we could avoid having to manage edit mode at all?